src/main/java/uk/gov/hmcts/darts/cases/controller/CaseController.java

```diff
@@ -70,33 +70,18 @@ public ResponseEntity<List<ScheduledCase>> casesGet(@NotNull @Parameter(name = "
-        @Size(max = 100) @Parameter(name = "event_text_contains", description = "Full or partial event_text", in = ParameterIn.QUERY) @Valid @RequestParam(value = "event_text_contains", required = false) String eventTextContains
-    ) {
-
-        GetCasesSearchRequest request = GetCasesSearchRequest.builder()
-            .caseNumber(StringUtils.trimToNull(caseNumber))
-            .courthouse(StringUtils.trimToNull(courthouse))
-            .courtroom(StringUtils.trimToNull(courtroom))
-            .judgeName(StringUtils.trimToNull(judgeName))
-            .defendantName(StringUtils.trimToNull(defendantName))
-            .dateFrom(dateFrom)
-            .dateTo(dateTo)
-            .eventTextContains(StringUtils.trimToNull(eventTextContains))
-            .build();
+        @Size(max = 20) @Parameter(name = "case_number", description = "Full or partial Case Number", in = ParameterIn.QUERY) @Valid @RequestParam(value = "case_number", required = false) String caseNumber, @Size(max = 30) @Parameter(name = "courthouse", description = "Full or partial Courthouse name", in = ParameterIn.QUERY) @Valid @RequestParam(value = "courthouse", required = false) String courthouse, @Size(max = 30) @Parameter(name = "courtroom", description = "Full or partial Courtroom name", in = ParameterIn.QUERY) @Valid @RequestParam(value = "courtroom", required = false) String courtroom, @Size(max = 30) @Parameter(name = "judge_name", description = "Full or partial Judge name", in = ParameterIn.QUERY) @Valid @RequestParam(value = "judge_name", required = false) String judgeName, @Size(max = 30) @Parameter(name = "defendant_name", description = "Full or partial Defendant name", in = ParameterIn.QUERY) @Valid @RequestParam(value = "defendant_name", required = false) String defendantName, @Parameter(name = "date_from", description = "DateFrom to search for the hearings in.", in = ParameterIn.QUERY) @Valid @RequestParam(value = "date_from", required = false) @DateTimeFormat(iso = DateTimeFormat.ISO.DATE) LocalDate dateFrom, @Parameter(name = "date_to", description = "DateTo to search for the hearings in.", in = ParameterIn.QUERY) @Valid @RequestParam(value = "date_to", required = false) @DateTimeFormat(iso = DateTimeFormat.ISO.DATE) LocalDate dateTo, @Size(max = 100) @Parameter(name = "event_text_contains", description = "Full or partial event_text", in = ParameterIn.QUERY) @Valid @RequestParam(value = "event_text_contains", required = false) String eventTextContains) {
+
+        GetCasesSearchRequest request = GetCasesSearchRequest.builder().caseNumber(StringUtils.trimToNull(caseNumber)).courthouse(
+            StringUtils.trimToNull(courthouse)).courtroom(StringUtils.trimToNull(courtroom)).judgeName(StringUtils.trimToNull(
+            judgeName)).defendantName(StringUtils.trimToNull(defendantName)).dateFrom(dateFrom).dateTo(dateTo).eventTextContains(
+            StringUtils.trimToNull(eventTextContains)).build();
```

Comment:
this used to be quite neatly organised, but seems to have been merged , can you fix?